.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
Maybe I'd structure it just like our youtrack issues - so the person reads the sectiona header, removes the body and writes his own body. So let's put something, so the user doesn't need to write a header

```suggestion
## Bug description
```